src/w2form.js

```diff
@@ -1032,8 +1032,6 @@
                     case 'email':
                     case 'pass':
                     case 'password':
-                        field.el.value = value;
-                        break;
```

Comment:
I add the class w2ui-input through the w2field init function, the line 1036 cause the above type field bypass w2field.init function.